docs/en/ingest-management/elastic-agent/running-on-kubernetes-gke-standard-managed-by-fleet.asciidoc

```diff
@@ -0,0 +1,170 @@
+[[running-on-kubernetes-gke-standard-managed-by-fleet]]
+= Run {agent} on GKE standard managed by {fleet}
+
+Use {agent} https://www.docker.elastic.co/r/beats/elastic-agent[Docker images] on Kubernetes to
+retrieve cluster metrics.
+
+TIP: Running {ecloud} on Kubernetes? Refer to {eck-ref}/k8s-elastic-agent-fleet.html[Run {elastic-agent} on ECK].
+
+ifeval::["{release-state}"=="unreleased"]
+
+However, version {version} of {agent} has not yet been
+released, so no Docker image is currently available for this version.
+
+endif::[]
+
+[discrete]
+== Prerequisites
+
+All data_streams with the `state_` prefix require `hosts` field pointing to `kube-state-metrics`
```

Comment:
I think that at this point of the documentation it might be not yet clear about which data_streams is it going, and what is a `hosts`.
maybe it could be better to just say what is required and keep rest as an clarification, smth like:
```
- `kube-state-metrics` is running and accessible within the cluster.

`kube-state-metrics` is not deployed by default in Kubernetes GKE. In order to install `kube-state-metrics` follow
the instructions for its deployment https://github.com/kubernetes/kube-state-metrics#kubernetes-deployment[here].
```